packages/material-ui/test/typescript/components.spec.tsx

```diff
@@ -283,7 +283,7 @@ const DialogTest = () => {
         </List>
       </div>
       <DialogContent>
-        <DialogContentText variant='body2' color='primary'>
```

Comment:
Same here. This change is not related to this PR. It was changed automatically by Prettier.